examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -367,6 +375,23 @@ def main():
 
     config.update({"forced_decoder_ids": model_args.forced_decoder_ids, "suppress_tokens": model_args.suppress_tokens})
 
+    # SpecAugment for whisper models
+    if getattr(config, "model_type", None) == "whisper":
+        config.update({"apply_spec_augment": model_args.apply_spec_augment})
+        # In order to keep the example easy to understand and to customize, we don't provide all the arguments of SpecAugment in ModelArguments
+        # You can customize the default values here as you want
+        if config.apply_spec_augment:
+            config.update(
+                {
+                    "mask_time_prob": 0.05,
+                    "mask_time_length": 10,
+                    "mask_time_min_masks": 2,
+                    "mask_feature_prob": 0.0,
+                    "mask_feature_length": 10,
+                    "mask_feature_min_masks": 0,
+                }
+            )
+
```

Comment:
Yeah! I was hesitating between not hard coding and giving some tips about these values! Totally agree with you! 

But should we remove the condition `getattr(config, "model_type", None) == "whisper"`? Not sure if it will confuse other models

Reply:
Cool! Let's keep in the `== "whisper"` condition then and remove the rest!